home-manager/fish.nix

```diff
@@ -47,6 +47,7 @@
 
       set -g fish_greeting
 
+      source "${../dependencies/podman/completions.fish}"
```

Comment:
fish made be much slower... 🤔 

```console
> bench_shells
Benchmark 1: /nix/store/am0wjwz25yw7q8llxhlbkh44jsan1a9m-zsh-5.9/bin/zsh --interactive -c exit
  Time (mean ± σ):      56.2 ms ±   1.6 ms    [User: 53.0 ms, System: 9.0 ms]
  Range (min … max):    54.1 ms …  58.0 ms    5 runs

Benchmark 2: /nix/store/vmgwn3xsbyj860z2bazvg4n10c4kp3xk-bash-interactive-5.2p26/bin/bash -i -c exit
  Time (mean ± σ):      42.1 ms ±   1.0 ms    [User: 41.8 ms, System: 9.7 ms]
  Range (min … max):    40.4 ms …  43.1 ms    5 runs

Benchmark 3: /nix/store/6m8rdk5zfdw7gnpisy942my0lw9my71z-fish-3.7.0/bin/fish --interactive --command exit
  Time (mean ± σ):     155.9 ms ±  10.8 ms    [User: 42.4 ms, System: 39.3 ms]
  Range (min … max):   140.2 ms … 165.7 ms    5 runs

Summary
  /nix/store/vmgwn3xsbyj860z2bazvg4n10c4kp3xk-bash-interactive-5.2p26/bin/bash -i -c exit ran
    1.34 ± 0.05 times faster than /nix/store/am0wjwz25yw7q8llxhlbkh44jsan1a9m-zsh-5.9/bin/zsh --interactive -c exit
    3.71 ± 0.27 times faster than /nix/store/6m8rdk5zfdw7gnpisy942my0lw9my71z-fish-3.7.0/bin/fish --interactive --command exit
```

Reply:
fixed with https://github.com/kachick/dotfiles/pull/423/commits/c0e9ec6707c85dc70d4c0112eff1492f6f75429b 🤷‍♂️